enterprise/app/workflows/workflows.tsx

```diff
@@ -326,6 +441,39 @@ class RepoItem extends React.Component<RepoItemProps, RepoItemState> {
                   <MenuItem onClick={this.onClickCopyWebhookUrl.bind(this)}>Copy webhook URL</MenuItem>
                 )}
                 <MenuItem onClick={this.onClickUnlinkMenuItem.bind(this)}>Unlink repository</MenuItem>
+                {/* The Run Workflow button is only supported for workflows configured with the Github App, not legacy workflows */}
+                {!this.props.webhookUrl && (
+                  <MenuItem
+                    onClick={this.onClickRunWorkflowMenuItem.bind(this)}
+                    disabled={this.state.isWorkflowRunning}>
+                    Run workflow
+                  </MenuItem>
+                )}
+                <Popup
```

Comment:
Rather than double-nesting popups (which I assume shows two shades, too?), could you just inline this to expand inside of the popup that we're already showing?

Reply:
I double nested the popups because I thought it would look weird to just append the 'run workflow' input form onto the existing menu. Here's what it looks like with the double popups: https://share.getcloudapp.com/xQuRXQ4r

Another option would be to move 'Run Workflow' out of the menu and into its own button

Reply:
Yeah, pulling it out of this menu sounds nice--it would match the UI that we have to re-trigger on an individual workflow page: 
![image](https://github.com/buildbuddy-io/buildbuddy/assets/10570856/9dc41084-8705-40ab-87e6-324252cb2036)